### ISSUE TITLE
Questions about fine-tuning Evo

### ISSUE THREAD

**Issue by HelloWorldLTY** [ArcInstitute/evo2#123]
Hi, thanks for your great work. I am working on a project which requires fine-tuning Evo2. However, after investigating the forward function of Evo2, it seems that the torch.no_grad() is set by default:

        try:
            # Original forward pass
            with torch.no_grad():
                logits = self.model.forward(input_ids)
            
            if return_embeddings:
                return logits, embeddings
            return logits, None

Does this mean we need to remove it for fine-tuning, or there exist other recommendation methods to perform fine-tuning? Thanks.

**Comment by HelloWorldLTY** [comment]
Also, there is no parameters() key in Evo2, so does this mean it is hard to fine-tune it with pytorch training framework?
